src/cup1d/likelihood/lya_theory.py

```diff
@@ -12,12 +14,24 @@
 class Theory(object):
     """Translator between the likelihood object and the emulator. This object
     will map from a set of CAMB parameters directly to emulator calls, without
-    going through our Delta^2_\star parametrisation """
-
-    def __init__(self,zs,emulator=None,verbose=False,
-                    F_model_fid=None,T_model_fid=None,P_model_fid=None,
-                    include_metals=[],
-                    cosmo_fid=None,free_param_names=None):
+    going through our Delta^2_\star parametrisation"""
+
+    def __init__(
+        self,
+        zs,
+        emulator=None,
+        verbose=False,
+        F_model_fid=None,
+        T_model_fid=None,
+        P_model_fid=None,
+        z_star=3.0,
+        kp_kms=0.009,
+        include_metals=[],
+        cosmo_fid=None,
+        free_param_names=None,
+        sim_igm="mpg_central",
```

Comment:
@jchavesmontero - Should we rename `sim_igm` to make it clear that this is used to set the fiducial IGM models? Maybe `sim_igm_fid` or something like this? We should also clarify and document the relation between this option and passing fiducial IGM models (I believe this `sim_igm` is only used if the others are not provided?)

Reply:
I renamed sim_igm to fid_igm before reading your comment. Do you think this is fine?